mlir/include/mlir/Dialect/SparseTensor/IR/Enums.h

```diff
@@ -154,9 +154,10 @@ enum class Action : uint32_t {
 
 /// This enum defines all the sparse representations supportable by
 /// the SparseTensor dialect. We use a lightweight encoding to encode
-/// both the "format" per se (dense, compressed, singleton, loose_compressed,
-/// two-out-of-four) as well as the "properties" (ordered, unique). The
-/// encoding is chosen for performance of the runtime library, and thus may
+/// the "format" per se (dense, compressed, singleton, loose_compressed,
+/// n-out-of-m), the "properties" (ordered, unique) as well as n and m for
```

Comment:
you mention n-out-of-m twice?

remove the "as well as ..." part

Reply:
The first n-out-of-m is the format, the "as well as n and m" are the actual `n` and `m`. This sentence is explaining what the encoding is: level format, level property, n and m.

Reply:
can you spell the second part out a bit more, for example,

as well as the value for n and m when the format is ....

Reply:
Sounds good!